apps/desktop/src/renderer/components/github/GitHubAppInstallPanel.tsx

```
@@ -57,11 +79,20 @@ export function GitHubAppInstallPanel({ variant = "settings" }: GitHubAppInstall
         error: error instanceof Error ? error.message : String(error),
       });
     } finally {
+      if (!mountedRef.current || statusRequestSeqRef.current !== requestSeq) return;
```

Biome (2.5.1)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)